src/work-allocation-2/containers/find-person-container/find-person-container.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { Person } from '../../models/dtos';
+
+@Component({
+    selector: 'exui-find-person-container',
+    templateUrl: './find-person-container.component.html'
+  })
+
+export class FindPersonContainerComponent {
+  public selectedPerson(person?: Person) {
+    console.log(person);
```

Comment:
console.log?

Reply:
I will delete this component FindPersonContainerComponent once @sreekanthpuligadda  finishes his testing. This is only added to test the selected item is identified correctly

Reply:
as agreed this will be deleted before merging